diff --git a/node_modules/@fortawesome/free-solid-svg-icons/faAllergies.js b/node_modules/@fortawesome/free-solid-svg-icons/faAllerg.mjs
similarity index 100%
rename from node_modules/@fortawesome/free-solid-svg-icons/faAllergies.js
rename to node_modules/@fortawesome/free-solid-svg-icons/faAllerg.mjs
diff --git a/node_modules/@fortawesome/free-solid-svg-icons/faBoxes.js b/node_modules/@fortawesome/free-solid-svg-icons/faBo.mjs
similarity index 100%
rename from node_modules/@fortawesome/free-solid-svg-icons/faBoxes.js
rename to node_modules/@fortawesome/free-solid-svg-icons/faBo.mjs
diff --git a/node_modules/@fortawesome/free-solid-svg-icons/faCakeCandles.js b/node_modules/@fortawesome/free-solid-svg-icons/faCakeCand.mjs
similarity index 100%
rename from node_modules/@fortawesome/free-solid-svg-icons/faCakeCandles.js
rename to node_modules/@fortawesome/free-solid-svg-icons/faCakeCand.mjs
diff --git a/node_modules/@fortawesome/free-solid-svg-icons/faCalendarTimes.js b/node_modules/@fortawesome/free-solid-svg-icons/faCalendarTi.mjs
similarity index 100%
rename from node_modules/@fortawesome/free-solid-svg-icons/faCalendarTimes.js
rename to node_modules/@fortawesome/free-solid-svg-icons/faCalendarTi.mjs
diff --git a/node_modules/@fortawesome/free-solid-svg-icons/faCapsules.js b/node_modules/@fortawesome/free-solid-svg-icons/faCapsu.mjs
similarity index 100%
rename from node_modules/@fortawesome/free-solid-svg-icons/faCapsules.js
rename to node_modules/@fortawesome/free-solid-svg-icons/faCapsu.mjs
diff --git a/node_modules/@fortawesome/free-solid-svg-icons/faChampagneGlasses.js b/node_modules/@fortawesome/free-solid-svg-icons/faChampagneGlas.mjs
similarity index 100%
rename from node_modules/@fortawesome/free-solid-svg-icons/faChampagneGlasses.js
rename to node_modules/@fortawesome/free-solid-svg-icons/faChampagneGlas.mjs
diff --git a/node_modules/@fortawesome/free-solid-svg-icons/faCircleNodes.js b/node_modules/@fortawesome/free-solid-svg-icons/faCircleNo.mjs
similarity index 100%
rename from node_modules/@fortawesome/free-solid-svg-icons/faCircleNodes.js
rename to node_modules/@fortawesome/free-solid-svg-icons/faCircleNo.mjs
diff --git a/node_modules/@fortawesome/free-solid-svg-icons/faCubes.js b/node_modules/@fortawesome/free-solid-svg-icons/faCu.mjs
similarity index 100%
rename from node_modules/@fortawesome/free-solid-svg-icons/faCubes.js
rename to node_modules/@fortawesome/free-solid-svg-icons/faCu.mjs
diff --git a/node_modules/@fortawesome/free-solid-svg-icons/faDiagnoses.js b/node_modules/@fortawesome/free-solid-svg-icons/faDiagno.mjs
similarity index 100%
rename from node_modules/@fortawesome/free-solid-svg-icons/faDiagnoses.js
rename to node_modules/@fortawesome/free-solid-svg-icons/faDiagno.mjs
diff --git a/node_modules/@fortawesome/free-solid-svg-icons/faFaceRollingEyes.js b/node_modules/@fortawesome/free-solid-svg-icons/faFaceRollingE.mjs
similarity index 100%
rename from node_modules/@fortawesome/free-solid-svg-icons/faFaceRollingEyes.js
rename to node_modules/@fortawesome/free-solid-svg-icons/faFaceRollingE.mjs
diff --git a/node_modules/@fortawesome/free-solid-svg-icons/faFileLines.js b/node_modules/@fortawesome/free-solid-svg-icons/faFileLi.mjs
similarity index 100%
rename from node_modules/@fortawesome/free-solid-svg-icons/faFileLines.js
rename to node_modules/@fortawesome/free-solid-svg-icons/faFileLi.mjs
diff --git a/node_modules/@fortawesome/free-solid-svg-icons/faGlasses.js b/node_modules/@fortawesome/free-solid-svg-icons/faGlas.mjs
similarity index 100%
rename from node_modules/@fortawesome/free-solid-svg-icons/faGlasses.js
rename to node_modules/@fortawesome/free-solid-svg-icons/faGlas.mjs
diff --git a/node_modules/@fortawesome/free-solid-svg-icons/faGripLines.js b/node_modules/@fortawesome/free-solid-svg-icons/faGripLi.mjs
similarity index 100%
rename from node_modules/@fortawesome/free-solid-svg-icons/faGripLines.js
rename to node_modules/@fortawesome/free-solid-svg-icons/faGripLi.mjs
diff --git a/node_modules/@fortawesome/free-solid-svg-icons/faHandSparkles.js b/node_modules/@fortawesome/free-solid-svg-icons/faHandSpark.mjs
similarity index 100%
rename from node_modules/@fortawesome/free-solid-svg-icons/faHandSparkles.js
rename to node_modules/@fortawesome/free-solid-svg-icons/faHandSpark.mjs
diff --git a/node_modules/@fortawesome/free-solid-svg-icons/faHandsBubbles.js b/node_modules/@fortawesome/free-solid-svg-icons/faHandsBubb.mjs
similarity index 100%
rename from node_modules/@fortawesome/free-solid-svg-icons/faHandsBubbles.js
rename to node_modules/@fortawesome/free-solid-svg-icons/faHandsBubb.mjs
diff --git a/node_modules/@fortawesome/free-solid-svg-icons/faHeadphones.js b/node_modules/@fortawesome/free-solid-svg-icons/faHeadpho.mjs
similarity index 100%
rename from node_modules/@fortawesome/free-solid-svg-icons/faHeadphones.js
rename to node_modules/@fortawesome/free-solid-svg-icons/faHeadpho.mjs
diff --git a/node_modules/@fortawesome/free-solid-svg-icons/faIcicles.js b/node_modules/@fortawesome/free-solid-svg-icons/faIcic.mjs
similarity index 100%
rename from node_modules/@fortawesome/free-solid-svg-icons/faIcicles.js
rename to node_modules/@fortawesome/free-solid-svg-icons/faIcic.mjs
diff --git a/node_modules/@fortawesome/free-solid-svg-icons/faImages.js b/node_modules/@fortawesome/free-solid-svg-icons/faIma.mjs
similarity index 100%
rename from node_modules/@fortawesome/free-solid-svg-icons/faImages.js
rename to node_modules/@fortawesome/free-solid-svg-icons/faIma.mjs
diff --git a/node_modules/@fortawesome/free-solid-svg-icons/faListSquares.js b/node_modules/@fortawesome/free-solid-svg-icons/faListSqua.mjs
similarity index 100%
rename from node_modules/@fortawesome/free-solid-svg-icons/faListSquares.js
rename to node_modules/@fortawesome/free-solid-svg-icons/faListSqua.mjs
diff --git a/node_modules/@fortawesome/free-solid-svg-icons/faMagicWandSparkles.js b/node_modules/@fortawesome/free-solid-svg-icons/faMagicWandSpark.mjs
similarity index 100%
rename from node_modules/@fortawesome/free-solid-svg-icons/faMagicWandSparkles.js
rename to node_modules/@fortawesome/free-solid-svg-icons/faMagicWandSpark.mjs
diff --git a/node_modules/@fortawesome/free-solid-svg-icons/faMehRollingEyes.js b/node_modules/@fortawesome/free-solid-svg-icons/faMehRollingE.mjs
similarity index 100%
rename from node_modules/@fortawesome/free-solid-svg-icons/faMehRollingEyes.js
rename to node_modules/@fortawesome/free-solid-svg-icons/faMehRollingE.mjs
diff --git a/node_modules/@fortawesome/free-solid-svg-icons/faMicrophoneLines.js b/node_modules/@fortawesome/free-solid-svg-icons/faMicrophoneLi.mjs
similarity index 100%
rename from node_modules/@fortawesome/free-solid-svg-icons/faMicrophoneLines.js
rename to node_modules/@fortawesome/free-solid-svg-icons/faMicrophoneLi.mjs
diff --git a/node_modules/@fortawesome/free-solid-svg-icons/faProcedures.js b/node_modules/@fortawesome/free-solid-svg-icons/faProcedu.mjs
similarity index 100%
rename from node_modules/@fortawesome/free-solid-svg-icons/faProcedures.js
rename to node_modules/@fortawesome/free-solid-svg-icons/faProcedu.mjs
diff --git a/node_modules/@fortawesome/free-solid-svg-icons/faRectangleTimes.js b/node_modules/@fortawesome/free-solid-svg-icons/faRectangleTi.mjs
similarity index 100%
rename from node_modules/@fortawesome/free-solid-svg-icons/faRectangleTimes.js
rename to node_modules/@fortawesome/free-solid-svg-icons/faRectangleTi.mjs
diff --git a/node_modules/@fortawesome/free-solid-svg-icons/faRoadSpikes.js b/node_modules/@fortawesome/free-solid-svg-icons/faRoadSpi.mjs
similarity index 100%
rename from node_modules/@fortawesome/free-solid-svg-icons/faRoadSpikes.js
rename to node_modules/@fortawesome/free-solid-svg-icons/faRoadSpi.mjs
diff --git a/node_modules/@fortawesome/free-solid-svg-icons/faShapes.js b/node_modules/@fortawesome/free-solid-svg-icons/faSha.mjs
similarity index 100%
rename from node_modules/@fortawesome/free-solid-svg-icons/faShapes.js
rename to node_modules/@fortawesome/free-solid-svg-icons/faSha.mjs
diff --git a/node_modules/@fortawesome/free-solid-svg-icons/faShareNodes.js b/node_modules/@fortawesome/free-solid-svg-icons/faShareNo.mjs
similarity index 100%
rename from node_modules/@fortawesome/free-solid-svg-icons/faShareNodes.js
rename to node_modules/@fortawesome/free-solid-svg-icons/faShareNo.mjs
diff --git a/node_modules/@fortawesome/free-solid-svg-icons/faSkullCrossbones.js b/node_modules/@fortawesome/free-solid-svg-icons/faSkullCrossbo.mjs
similarity index 100%
rename from node_modules/@fortawesome/free-solid-svg-icons/faSkullCrossbones.js
rename to node_modules/@fortawesome/free-solid-svg-icons/faSkullCrossbo.mjs
diff --git a/node_modules/@fortawesome/free-solid-svg-icons/faSprayCanSparkles.js b/node_modules/@fortawesome/free-solid-svg-icons/faSprayCanSpark.mjs
similarity index 100%
rename from node_modules/@fortawesome/free-solid-svg-icons/faSprayCanSparkles.js
rename to node_modules/@fortawesome/free-solid-svg-icons/faSprayCanSpark.mjs
diff --git a/node_modules/@fortawesome/free-solid-svg-icons/faSquareShareNodes.js b/node_modules/@fortawesome/free-solid-svg-icons/faSquareShareNo.mjs
similarity index 100%
rename from node_modules/@fortawesome/free-solid-svg-icons/faSquareShareNodes.js
rename to node_modules/@fortawesome/free-solid-svg-icons/faSquareShareNo.mjs
diff --git a/node_modules/@fortawesome/free-solid-svg-icons/faTimes.js b/node_modules/@fortawesome/free-solid-svg-icons/faTi.mjs
similarity index 100%
rename from node_modules/@fortawesome/free-solid-svg-icons/faTimes.js
rename to node_modules/@fortawesome/free-solid-svg-icons/faTi.mjs
diff --git a/node_modules/@fortawesome/free-solid-svg-icons/faUserTimes.js b/node_modules/@fortawesome/free-solid-svg-icons/faUserTi.mjs
similarity index 100%
rename from node_modules/@fortawesome/free-solid-svg-icons/faUserTimes.js
rename to node_modules/@fortawesome/free-solid-svg-icons/faUserTi.mjs
diff --git a/node_modules/@fortawesome/free-solid-svg-icons/faUsersBetweenLines.js b/node_modules/@fortawesome/free-solid-svg-icons/faUsersBetweenLi.mjs
similarity index 100%
rename from node_modules/@fortawesome/free-solid-svg-icons/faUsersBetweenLines.js
rename to node_modules/@fortawesome/free-solid-svg-icons/faUsersBetweenLi.mjs
diff --git a/node_modules/@fortawesome/free-solid-svg-icons/faVestPatches.js b/node_modules/@fortawesome/free-solid-svg-icons/faVestPatc.mjs
similarity index 100%
rename from node_modules/@fortawesome/free-solid-svg-icons/faVestPatches.js
rename to node_modules/@fortawesome/free-solid-svg-icons/faVestPatc.mjs
diff --git a/node_modules/@fortawesome/free-solid-svg-icons/faViruses.js b/node_modules/@fortawesome/free-solid-svg-icons/faViru.mjs
similarity index 100%
rename from node_modules/@fortawesome/free-solid-svg-icons/faViruses.js
rename to node_modules/@fortawesome/free-solid-svg-icons/faViru.mjs
diff --git a/node_modules/@fortawesome/free-solid-svg-icons/faVolumeTimes.js b/node_modules/@fortawesome/free-solid-svg-icons/faVolumeTi.mjs
similarity index 100%
rename from node_modules/@fortawesome/free-solid-svg-icons/faVolumeTimes.js
rename to node_modules/@fortawesome/free-solid-svg-icons/faVolumeTi.mjs
diff --git a/node_modules/@fortawesome/free-solid-svg-icons/faWandMagicSparkles.js b/node_modules/@fortawesome/free-solid-svg-icons/faWandMagicSpark.mjs
similarity index 100%
rename from node_modules/@fortawesome/free-solid-svg-icons/faWandMagicSparkles.js
rename to node_modules/@fortawesome/free-solid-svg-icons/faWandMagicSpark.mjs
diff --git a/node_modules/@fortawesome/free-solid-svg-icons/faWandSparkles.js b/node_modules/@fortawesome/free-solid-svg-icons/faWandSpark.mjs
similarity index 100%
rename from node_modules/@fortawesome/free-solid-svg-icons/faWandSparkles.js
rename to node_modules/@fortawesome/free-solid-svg-icons/faWandSpark.mjs
diff --git a/node_modules/@fortawesome/free-solid-svg-icons/faXmarksLines.js b/node_modules/@fortawesome/free-solid-svg-icons/faXmarksLi.mjs
similarity index 100%
rename from node_modules/@fortawesome/free-solid-svg-icons/faXmarksLines.js
rename to node_modules/@fortawesome/free-solid-svg-icons/faXmarksLi.mjs
diff --git a/node_modules/@fortawesome/free-solid-svg-icons/index.es.js b/node_modules/@fortawesome/free-solid-svg-icons/index.mjs
similarity index 100%
rename from node_modules/@fortawesome/free-solid-svg-icons/index.es.js
rename to node_modules/@fortawesome/free-solid-svg-icons/index.mjs
diff --git a/node_modules/@fortawesome/free-solid-svg-icons/package.json b/node_modules/@fortawesome/free-solid-svg-icons/package.json
index 57c729d..2e543d3 100644
--- a/node_modules/@fortawesome/free-solid-svg-icons/package.json
+++ b/node_modules/@fortawesome/free-solid-svg-icons/package.json
@@ -26,24 +26,24 @@
   "version": "6.1.2",
   "name": "@fortawesome/free-solid-svg-icons",
   "main": "index.js",
-  "module": "index.es.js",
-  "jsnext:main": "index.es.js",
+  "module": "index.mjs",
+  "jsnext:main": "index.mjs",
   "license": "(CC-BY-4.0 AND MIT)",
   "types": "./index.d.ts",
   "sideEffects": false,
   "exports": {
     ".": {
-      "import": "./index.es.js",
+      "import": "./index.mjs",
       "require": "./index.js",
       "default": "./index.js"
     },
     "./index": {
-      "import": "./index.es.js",
+      "import": "./index.mjs",
       "require": "./index.js",
       "default": "./index.js"
     },
     "./index.js": {
-      "import": "./index.es.js",
+      "import": "./index.mjs",
       "require": "./index.js",
       "default": "./index.js"
     },
