diff --git a/node_modules/@fortawesome/free-regular-svg-icons/faCalendarTimes.js b/node_modules/@fortawesome/free-regular-svg-icons/faCalendarTi.mjs
similarity index 100%
rename from node_modules/@fortawesome/free-regular-svg-icons/faCalendarTimes.js
rename to node_modules/@fortawesome/free-regular-svg-icons/faCalendarTi.mjs
diff --git a/node_modules/@fortawesome/free-regular-svg-icons/faFaceRollingEyes.js b/node_modules/@fortawesome/free-regular-svg-icons/faFaceRollingE.mjs
similarity index 100%
rename from node_modules/@fortawesome/free-regular-svg-icons/faFaceRollingEyes.js
rename to node_modules/@fortawesome/free-regular-svg-icons/faFaceRollingE.mjs
diff --git a/node_modules/@fortawesome/free-regular-svg-icons/faFileLines.js b/node_modules/@fortawesome/free-regular-svg-icons/faFileLi.mjs
similarity index 100%
rename from node_modules/@fortawesome/free-regular-svg-icons/faFileLines.js
rename to node_modules/@fortawesome/free-regular-svg-icons/faFileLi.mjs
diff --git a/node_modules/@fortawesome/free-regular-svg-icons/faImages.js b/node_modules/@fortawesome/free-regular-svg-icons/faIma.mjs
similarity index 100%
rename from node_modules/@fortawesome/free-regular-svg-icons/faImages.js
rename to node_modules/@fortawesome/free-regular-svg-icons/faIma.mjs
diff --git a/node_modules/@fortawesome/free-regular-svg-icons/faMehRollingEyes.js b/node_modules/@fortawesome/free-regular-svg-icons/faMehRollingE.mjs
similarity index 100%
rename from node_modules/@fortawesome/free-regular-svg-icons/faMehRollingEyes.js
rename to node_modules/@fortawesome/free-regular-svg-icons/faMehRollingE.mjs
diff --git a/node_modules/@fortawesome/free-regular-svg-icons/faRectangleTimes.js b/node_modules/@fortawesome/free-regular-svg-icons/faRectangleTi.mjs
similarity index 100%
rename from node_modules/@fortawesome/free-regular-svg-icons/faRectangleTimes.js
rename to node_modules/@fortawesome/free-regular-svg-icons/faRectangleTi.mjs
diff --git a/node_modules/@fortawesome/free-regular-svg-icons/index.es.js b/node_modules/@fortawesome/free-regular-svg-icons/index.mjs
similarity index 100%
rename from node_modules/@fortawesome/free-regular-svg-icons/index.es.js
rename to node_modules/@fortawesome/free-regular-svg-icons/index.mjs
diff --git a/node_modules/@fortawesome/free-regular-svg-icons/package.json b/node_modules/@fortawesome/free-regular-svg-icons/package.json
index 46dc11b..5dd2914 100644
--- a/node_modules/@fortawesome/free-regular-svg-icons/package.json
+++ b/node_modules/@fortawesome/free-regular-svg-icons/package.json
@@ -26,24 +26,24 @@
   "version": "6.1.2",
   "name": "@fortawesome/free-regular-svg-icons",
   "main": "index.js",
-  "module": "index.es.js",
-  "jsnext:main": "index.es.js",
+  "module": "index.mjs",
+  "jsnext:main": "index.mjs",
   "license": "(CC-BY-4.0 AND MIT)",
   "types": "./index.d.ts",
   "sideEffects": false,
   "exports": {
     ".": {
-      "import": "./index.es.js",
+      "import": "./index.mjs",
       "require": "./index.js",
       "default": "./index.js"
     },
     "./index": {
-      "import": "./index.es.js",
+      "import": "./index.mjs",
       "require": "./index.js",
       "default": "./index.js"
     },
     "./index.js": {
-      "import": "./index.es.js",
+      "import": "./index.mjs",
       "require": "./index.js",
       "default": "./index.js"
     },
