diff --git a/node_modules/@fortawesome/free-brands-svg-icons/faFantasyFlightGames.js b/node_modules/@fortawesome/free-brands-svg-icons/faFantasyFlightGa.mjs
similarity index 100%
rename from node_modules/@fortawesome/free-brands-svg-icons/faFantasyFlightGames.js
rename to node_modules/@fortawesome/free-brands-svg-icons/faFantasyFlightGa.mjs
diff --git a/node_modules/@fortawesome/free-brands-svg-icons/faItunes.js b/node_modules/@fortawesome/free-brands-svg-icons/faItu.mjs
similarity index 100%
rename from node_modules/@fortawesome/free-brands-svg-icons/faItunes.js
rename to node_modules/@fortawesome/free-brands-svg-icons/faItu.mjs
diff --git a/node_modules/@fortawesome/free-brands-svg-icons/faPagelines.js b/node_modules/@fortawesome/free-brands-svg-icons/faPageli.mjs
similarity index 100%
rename from node_modules/@fortawesome/free-brands-svg-icons/faPagelines.js
rename to node_modules/@fortawesome/free-brands-svg-icons/faPageli.mjs
diff --git a/node_modules/@fortawesome/free-brands-svg-icons/faStripeS.js b/node_modules/@fortawesome/free-brands-svg-icons/faStri.mjs
similarity index 100%
rename from node_modules/@fortawesome/free-brands-svg-icons/faStripeS.js
rename to node_modules/@fortawesome/free-brands-svg-icons/faStri.mjs
diff --git a/node_modules/@fortawesome/free-brands-svg-icons/index.es.js b/node_modules/@fortawesome/free-brands-svg-icons/index.mjs
similarity index 100%
rename from node_modules/@fortawesome/free-brands-svg-icons/index.es.js
rename to node_modules/@fortawesome/free-brands-svg-icons/index.mjs
diff --git a/node_modules/@fortawesome/free-brands-svg-icons/package.json b/node_modules/@fortawesome/free-brands-svg-icons/package.json
index c3b0140..15d0255 100644
--- a/node_modules/@fortawesome/free-brands-svg-icons/package.json
+++ b/node_modules/@fortawesome/free-brands-svg-icons/package.json
@@ -26,24 +26,24 @@
   "version": "6.1.2",
   "name": "@fortawesome/free-brands-svg-icons",
   "main": "index.js",
-  "module": "index.es.js",
-  "jsnext:main": "index.es.js",
+  "module": "index.mjs",
+  "jsnext:main": "index.mjs",
   "license": "(CC-BY-4.0 AND MIT)",
   "types": "./index.d.ts",
   "sideEffects": false,
   "exports": {
     ".": {
-      "import": "./index.es.js",
+      "import": "./index.mjs",
       "require": "./index.js",
       "default": "./index.js"
     },
     "./index": {
-      "import": "./index.es.js",
+      "import": "./index.mjs",
       "require": "./index.js",
       "default": "./index.js"
     },
     "./index.js": {
-      "import": "./index.es.js",
+      "import": "./index.mjs",
       "require": "./index.js",
       "default": "./index.js"
     },
